### PR TITLE
Fix: Portfolio site leads to 404 

### LINKED ISSUE #147: Portfolio site leads to 404
**Yikes! Briefly, what seems wrong?**
When I click on "Portfolio", I am taken to the url https://cherryon.tech/bio/[users name]/[portfolio website url], page says "404 not found" 

**Expected Behavior**
You should be taken to the portfolio website

**Current Behavior**
Get 404 not found error 
image

**Possible Solution**
Any ideas how to fix it? Tell us here!

**Steps to Reproduce**
1. Go to https://cherryon.tech/about/
2. Click on the word, "site" for a card that has **site** on the lower right hand corner

### PR DESCRIPTION
**Pull request by TishG**
## This PR fixes...

This PR fixes #147. Appends 'https://' to the urls that do not have 'http://' or 'https://' so that the link navigates to the site instead of 404 error page.

## What I did...

- add libraries folder and utils file
- add util function for validating and mutating the url
- pass all urls into util function

## How to test...

1. Navigate to http://localhost:8080/about/
2. Click 'site' for each card that contains a site button
3. Verify that it navigates to their website and not a 404 error page
4. Navigate to the bio page (ex: http://localhost:8080/bio/julieth-fajardo/) for each card that contains a site button 
5. Click on the 'Portfolio' link
6. Verify that it navigates to their website and not a 404 error page

### CODE REVIEW

**Comment by TishG** [comment]
@novellac 
I would like you to take a look at the util function to see if it covers all the boundaries. I didn't feel it was necessary to check that the url is a string since the author is adding in a url that we are assuming will be a string with a .com at the end. 

The next step is to find where to wrap the url with this function once you approve of the function.

File: src/pages/About.vue
Comment (novellac) [inline src/pages/About.vue:104]: Suggestion: Remove these unused methods.

File: src/templates/Biopost.vue
Comment (novellac) [inline src/templates/Biopost.vue:19]: I think here we still want the email to be dynamic, so keep the `:` on the `href`.

File: src/libraries/utils.js
Comment (novellac) [inline src/libraries/utils.js:14]: Compliment: I like your explanation here, it gives great context.

**Review by novellac — APPROVED** [review]
Great job! Approved, pending resolution of the comments below.

**Pull request closed without merging**
